packages/mdc-toolbar/_variables.scss

```diff
@@ -17,8 +17,12 @@
 $mdc-toolbar-row-height: 64px;
 $mdc-toolbar-mobile-row-height: 56px;
 $mdc-toolbar-mobile-landscape-row-height: 48px;
-$mdc-toolbar-padding: 20px 28px;
-$mdc-toobar-element-vertical-padding: 16px;
+
+$mdc-toolbar-element-vertical-padding: 16px;
+$mdc-toolbar-element-horizontal-padding-desktop: 24px;
+$mdc-toolbar-element-horizontal-padding-mobile: 16px;
+$mdc-toolbar-title-margin-to-icon-menu-desktop: 8px;
```

Comment:
Nit: would these names read better as `title-margin-to-menu-icon` (rather than `icon-menu`)?

Reply:
It seems this won't introduce confusion with our modifier class name.